docs/en/observability/cloud-monitoring/aws/monitor-aws-cloudwatch-firehose.asciidoc

```diff
@@ -0,0 +1,270 @@
+
+You should see something similar:
+
+image::firehose-cloudwatch-sample-logs.png[CloudWatch log group with sample logs]
+
+Take note of the log group name for this Lambda function, and move to the next section.
+
+[discrete]
+[[firehose-cloudwatch-step-three]]
+== Step 3: Create a stream in Amazon Data Firehose
+
+. Go to the https://console.aws.amazon.com/[AWS console] and navigate to Amazon Data Firehose.  
+
+. Click *Create Firehose stream* and choose the source and destination of your Firehose stream. Unless you are streaming data from Kinesis Data Streams, set source to `Direct PUT` and destination to `Elastic`. 
+
+. Provide a meaningful *Firehose stream name* that will allow you to identify this delivery stream later. 
+
+NOTE: For advanced use cases, source records can be transformed by invoking a custom Lambda function. When using Elastic integrations, this should not be required.
+
+Set the **Parameters** in the **Destination settings** section.
+
+[discrete]
+[[firehose-cloudwatch-step-three-desination-settings-parameters]]
+=== Parameters
+
+Use the following parameters:
+
+
+[cols="1,1",options="header"]
+|===
+|Name
+|Value
+
+| `es_datastream_name`
+| `logs-aws.generic-default`
+
+|===
+
+[discrete]
+[[firehose-cloudwatch-step-four]]
+== Step 4: Set up a subscription filter to send Lambda function log events to a Firehose stream
+
+The Firehose stream is ready to send logs to our Elastic Cloud deployment.
+
+Next steps:
+
+- Open the log group with the Lambda function log events
+- Create a subscription filter for Amazon Data Firehose 
+
+[discrete]
+[[firehose-cloudwatch-step-four-log-group]]
+=== Open the log group with the Lambda function log events
+
+Please open the log group where the Lambda service is sending the events. We must forward these events to an Elastic stack using the Firehose delivery stream.
+
+The CloudWatch log group offers a https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Subscriptions.html[subscription filter]. The subscription filter allows users to pick log events from the log group and forward them to other services, such as an Amazon Kinesis stream, an Amazon Data Firehose stream, or AWS Lambda.
+
+[discrete]
+[[firehose-cloudwatch-step-four-subscription-filter]]
+=== Create a subscription filter for Amazon Data Firehose 
+
+- Choose destination
+- Grant permission
+- Configure log format and filters
+
+[discrete]
+[[firehose-cloudwatch-step-four-subscription-filter-destination]]
+==== Choose destination
+
+Please select the Firehose stream we create in the previous step.
+
+[discrete]
+[[firehose-cloudwatch-step-four-subscription-filter-permission]]
+==== Grant permission
+
+Grant the CloudWatch service to send log events to the stream in Firehose.
+
+This step consists of two parts::
+
+1. Create a new role with a trust policy that allows CloudWatch service to assume the role.
+2. Assign a policy to the role that permits "putting records" into a Firehose  stream.
```

Comment:
I don't think it hurts, but it's probably better to be consistent with the asciidoc list style for steps. Leading periods (.) are better because it means you don't have to manage the numbering manually.
```suggestion
. Create a new role with a trust policy that allows CloudWatch service to assume the role.
. Assign a policy to the role that permits "putting records" into a Firehose  stream.
```